src/tests_deprecated/unit/engines/gna/graph_tools/graph_copy_tests.cpp

```diff
@@ -21,13 +21,39 @@ using namespace testing;
 using namespace InferenceEngine;
 using namespace std;
 using namespace GraphTest;
-
+using namespace ov::intel_gna::frontend;
 
 class GraphCopyTests : public GraphTestsBase {
 
 protected:
     MockCopier mc;
 
+#ifdef ENABLE_INTEL_GNA
```

Comment:
This conditional compilation had been present in the original file already, but I think it can be completely removed now, when the tests are sorted in plugins directories.